packages/api/lib/errors.js

```diff
@@ -32,10 +35,26 @@ class EarthdataLoginError extends Error {
   }
 }
 
+const resourceNotFoundInfo = 'Check if topic subscription and/or lambda trigger have been manually deleted from AWS. If so, rule may need to be manually disabled/deleted.';
```

Comment:
This warning may be too specific/directive.
```suggestion
const resourceNotFoundInfo = 'One solution may be to check if topic subscription and/or lambda trigger have been manually deleted from AWS. If so, rule may need to be manually disabled/deleted.';
```

Reply:
I'm saying this because I was naively editing a kinesis rule to be SNS, and got your error.  I of course can't reproduce it now.

Reply:
Yeah, there probably are more ways for this to crop up, so having it be less specific makes sense